modules/swagger-parser-v3/src/main/java/io/swagger/v3/parser/util/OpenAPIDeserializer.java

```diff
@@ -386,6 +405,15 @@ public OpenAPI parseRoot(JsonNode node, ParseResult result, String path) {
 		return openAPI;
 	}
 
+	private void validateReservedKeywords(Map<String, Set<String>> specKeys, String key, String location, ParseResult result) {
+		if(result.isOpenapi31() && specKeys.get("RESERVED_KEYWORDS").stream()
```

Comment:
better to add a new boolean flag in result, like `oaiInternal` or `oaiAuthor` to allow for any key prefix if the author is indeed OAI